tests/conftest.py

```
@@ -0,0 +1,129 @@
+from can import Message
+from uds.addressing import AddressingType
+from uds.can import CanAddressingFormat, CanAddressingInformation, CanDlcHandler, CanSegmenter
+from uds.can.addressing import (
+    AbstractCanAddressingInformation,
+    ExtendedCanAddressingInformation,
+    Mixed11BitCanAddressingInformation,
+    Mixed29BitCanAddressingInformation,
+    NormalCanAddressingInformation,
+    NormalFixedCanAddressingInformation,
+)
+from uds.message import UdsMessage
```

Ruff (0.12.2)

`uds.message.UdsMessage` imported but unused

Remove unused import: `uds.message.UdsMessage`

(F401)